apps/design-land/src/app/loading-icon/loading-icon.component.html

```diff
@@ -1,2 +1,33 @@
-<daff-loading-icon [diameter]="50" color="secondary"></daff-loading-icon>
-<daff-loading-icon [diameter]="50"></daff-loading-icon>
\ No newline at end of file
+<daff-article>
+  <h1 daffArticleTitle>Loading Icon</h1>
+  <p daffArticleLead>
+    The loading icon is used as an indicator of an event in progress.
+  </p>
+
+  <h2>About</h2>
+  <p>
+    This loading icon can be used to indicate that an event is ocurring and is still in progress.
+    The color and size of the loading icon can be customized to suit the needs of your project.
+  </p>
+
+  <h3>Diameter</h3>
+  <design-land-example-viewer example="loading-icon-diameter"></design-land-example-viewer>
```

Comment:
Add information about diameter property:

The diameter of a loading icon can be defined by using the `diameter` property. By default, the diameter is set to `60`.

Reply:
9b7b26bbc4fddb0ae5f3502223860f8fc249353a